app/src/main/java/com/example/henzoshimada/feeltrip/MoodAdapter.java

```diff
@@ -75,7 +80,9 @@ public View getView(int position, View convertView, ViewGroup parent) {
         Log.d("imageTag","new array list");
         viewHolder.userName.setText(mood.getUsername());
         viewHolder.date.setText(mood.getDate().toString());
-        viewHolder.description.setText(mood.getDescription());
+        viewHolder.description.setText(Html.fromHtml(mood.getDescription())); //TODO: This is depreciated, maybe replace?
+        viewHolder.append.setText(" -Feeling " + mood.getEmotionalState());
```

Comment:
Feeling should be append to the description. It will be more flexible and easier when we have to color code the - Feeling XXXX part